docs/source/docs/examples/aimingatatarget.md

```diff
@@ -1,16 +1,16 @@
 # Aiming at a Target
 
-The following example is from the PhotonLib example repository ([Java](https://github.com/PhotonVision/photonvision/tree/master/photonlib-java-examples/aimattarget)/[C++](https://github.com/PhotonVision/photonvision/tree/master/photonlib-cpp-examples/aimattarget)).
+The following example is from the PhotonLib example repository ([Java](https://github.com/PhotonVision/photonvision/tree/master/photonlib-java-examples/aimattarget)).
 
 ## Knowledge and Equipment Needed
 
 - Robot with a vision system running PhotonVision
-- Target
-- Ability to track a target by properly tuning a pipeline
+- AprilTag 7
+- Ability to track a AprilTags with a well-tuned pipeline
 
 ## Code
 
-Now that you have properly set up your vision system and have tuned a pipeline, you can now aim your robot/turret at the target using the data from PhotonVision. This data is reported over NetworkTables and includes: latency, whether there is a target detected or not, pitch, yaw, area, skew, and target pose relative to the robot. This data will be used/manipulated by our vendor dependency, PhotonLib. The documentation for the Network Tables API can be found {ref}`here <docs/additional-resources/nt-api:Getting Target Information>` and the documentation for PhotonLib {ref}`here <docs/programming/photonlib/adding-vendordep:What is PhotonLib?>`.
+Now that you have properly set up your vision system and have tuned a pipeline, you can now aim your robot/turret at an AprilTag using the data from PhotonVision. This data is reported over NetworkTables and includes: latency, whether there is a AprilTag detected or not, pitch, yaw, area, skew, and target pose relative to the robot. This data will be used/manipulated by the vendor dependency, PhotonLib. The documentation for the Network Tables API can be found {ref}`here <docs/additional-resources/nt-api:Getting Target Information>` and the documentation for PhotonLib {ref}`here <docs/programming/photonlib/adding-vendordep:What is PhotonLib?>`.
```

Comment:
Pedantic again but this is relative to your camera not your robot correct? I'd also err away from linking to the NT docs here and just push users towards the PhotonLib docs.

Reply:
I went ahead and updated the "required equipment" to be a bit more pedantic about how the camera must be rigidly mounted to the robot frame, and took out the turret reference. Hopefully... that's enough to allow "Robot" (what they care about) and "Camera" (what they're thinking about) to become interchangeable.

Reply:
and then I also deleted a bunch of stuff. why write many word when few word do trick